src/components/Dialog/BaseDialog/base-dialog.module.scss

```diff
@@ -8,7 +8,7 @@
         position: absolute;
         padding: $space-l-fitted;
         background: var(--background-color);
-        border-radius: $corner-radius-m;
+        border-radius: $corner-radius-xl;
```

Comment:
Theme 2 needs o be gated behind a global theme name

Reply:
Let's discuss the pros and cons of RI/FI via release branches vs theme using the existing ConfigProvider capabilities.